common/on_action/wc_lifestyles/wc_magic_lifestyles_on_actions.txt

```diff
@@ -0,0 +1,72 @@
+﻿magic_lifestyles_progress_pulse = { # Fires 4 times per year
+	trigger = {
```

Comment:
I suggest using this for the necromancy:
https://wowpedia.fandom.com/wiki/File:BTNAnimateDead-Reforged.png

I suggest using this for the shadow (because the current one gives me a lot of necromancy vibes):
https://wowpedia.fandom.com/wiki/File:BTNOrbOfDarkness-Reforged.png

IRRC, that's a reforged version of the icon you use for the fire:
https://wowpedia.fandom.com/wiki/File:BTNFire-Reforged.png

I suggest this for the arcane (because the current one gives me shadow and necromacy vibes):
https://wowpedia.fandom.com/wiki/File:BTNMagicalSentry-Reforged.png

I suggest this for the earth:
https://wowpedia.fandom.com/wiki/File:BTNEarthquake-Reforged.png

[Icon Sause](https://wowpedia.fandom.com/wiki/Wowpedia:Warcraft_III_icons/Reforged_icon_list)

Reply:
Done?